dagshub/common/helpers.py

```diff
@@ -0,0 +1,10 @@
+from dagshub.common import config
+import urllib
+import requests
+
+def get_default_branch(owner, reponame, auth):
+    res = requests.get(urllib.parse.urljoin(config.host, config.REPO_INFO_URL.format(
+        owner=owner,
```

Comment:
```suggestion
def get_default_branch(owner, reponame, auth, config=config.host):
    res = requests.get(urllib.parse.urljoin(host, config.REPO_INFO_URL.format(
        owner=owner,
```
We need to keep the ability of the host being overrideable.
I don't see a host arg in any of the users, but still